lib/taskana-core/src/test/java/acceptance/classification/UpdateClassificationAccTest.java

```diff
@@ -0,0 +1,68 @@
+package acceptance.classification;
+
+import static org.junit.Assert.assertEquals;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import org.h2.store.fs.FileUtils;
+import org.junit.AfterClass;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import acceptance.AbstractAccTest;
+import pro.taskana.Classification;
+import pro.taskana.ClassificationService;
+import pro.taskana.exceptions.ClassificationNotFoundException;
+
+/**
+ * Acceptance test for all "update classification" scenarios.
+ */
+public class UpdateClassificationAccTest extends AbstractAccTest {
+
+    public UpdateClassificationAccTest() {
+        super();
+    }
+
+    @Ignore
+    @Test
+    public void testUpdateClassification() throws SQLException, ClassificationNotFoundException {
+        ClassificationService classificationService = taskanaEngine.getClassificationService();
+        Classification classification = classificationService.getClassification("T2100", "DOMAIN_A");
+        classification.setApplicationEntryPoint("testApplicationEntryPoint");
```

Comment:
What does ApplicationEntryPoint does - what´s the usecase of it? Just question - it´s going to be integreated.